src/createServer.js

```diff
@@ -1,11 +1,28 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+const { UserModel } = require('./models/users');
+const { ExpenseModel } = require('./models/expenses');
+const { router: usersRouter } = require('./routes/users.js');
+const { router: expensesRouter } = require('./routes/expenses.js');
 
 function createServer() {
   // Use express to create a server
   // Add a routes to the server
   // Return the server (express app)
+
+  const app = express();
+
+  app.use(cors());
+
+  app.use('/users', express.json(), usersRouter);
+  app.use('/expenses', express.json(), expensesRouter);
+
+  UserModel.reset();
+  ExpenseModel.reset();
```

Comment:
The `reset()` methods for `UserModel` and `ExpenseModel` are called unconditionally. Ensure that this is intended, as it will reset the models every time the server starts, potentially leading to data loss. Consider adding a condition or configuration to control when these methods should be called.